packages/components/src/components/hds/advanced-table/index.ts

```diff
@@ -259,6 +268,12 @@ export default class HdsAdvancedTable extends Component<HdsAdvancedTableSignatur
     }
   }
 
+  get isEmpty(): boolean {
```

Comment:
Why are you using an isEmpty arg? Should we just check that the length of the model is 0?